docs/_includes/sidenav.html

```diff
@@ -88,7 +88,7 @@
     {% else %}
       {% assign elementsPos = elementsPosStack | last %}
       {% assign pos = posStack | last %}
-</li></ul></div>
+</ul></div></li>
```

Comment:
where are the starting tags for div and li?

Reply:
here: https://github.com/apache/flink/pull/9439/files/530bf78fdb79319e0a9f28e0fdf679559cee7c4c#diff-24bed5845471ba1bdda01fe23f23aee8R118